lib/vanagon/component/source/http.rb

```diff
@@ -105,7 +105,7 @@ def extract(tar)
             end
           else
             # Extension does not appear to be an archive
-            return nil
+            return ':'
```

Comment:
true is builtin as well. I guess the /bin is the problem, that tells it not to use the built in. 

    stahnma@stahnma-pro ~> type true
   true is a shell builtin